webplugin/js/app/kommunicate-client.js

```diff
@@ -94,7 +94,9 @@ Kommunicate.client={
                     if (typeof callback == 'function') {
                         callback(response.data.value);
                     }
-                    response && response.data && KommunicateUI.handleWaitingQueueMessage(response.data);
+                    CURRENT_GROUP_DATA.tabId = response.data.clientGroupId;
+                    CURRENT_GROUP_DATA.conversationStatus = response.data.metadata.CONVERSATION_STATUS;
```

Comment:
Can we add this check-in loadTab method itself?
Near below code
```
                    if (typeof MCK_ON_TAB_CLICKED === 'function') {
                        MCK_ON_TAB_CLICKED({
                            tabId: params.tabId,
                            isGroup: params.isGroup
                        });
                    }
```

Reply:
@Kusum-Dhoundiyal Changes were not made.
Please don't resolve the comment without resolving them.

Reply:
@ab14bhardwaj  On create group tab clicked will not get triggered that's why I am adding CURRENT_GROUP_DATA here . Without this real time waiting will not work .

Reply:
There is a check for the conditional working of loadTab function inside getGroup function.
Please add the above check-in getGroup function.